check-before-deployment.md

```diff
@@ -667,20 +667,22 @@ Take the following steps to check the current operating system configuration and
     ```bash
     echo "fs.file-max = 1000000">> /etc/sysctl.conf
     echo "net.core.somaxconn = 32768">> /etc/sysctl.conf
-    echo "net.ipv4.tcp_tw_recycle = 0">> /etc/sysctl.conf
     echo "net.ipv4.tcp_syncookies = 0">> /etc/sysctl.conf
     echo "vm.overcommit_memory = 1">> /etc/sysctl.conf
     echo "vm.min_free_kbytes = 1048576">> /etc/sysctl.conf
     sysctl -p
     ```
 
+    > **Warning:**
+    >
+    > It is not recommended to increase `vm.min_free_kbytes` on systems with less than 16 GiB of memory, because it might cause instability and boot failures.
```

Comment:
```suggestion
    > It is not recommended to increase the value of `vm.min_free_kbytes` on systems with less than 16 GiB of memory, because it might cause instability and boot failures.
```